code/modules/antagonists/cult/cult_comms.dm

```diff
@@ -79,6 +79,101 @@
 			var/link = FOLLOW_LINK(M, user)
 			to_chat(M, "[link] [my_message]")
 
+///a button for requesting cultists come to your location, lasts for 3 minutes (during which time no one else can request assistance)
+/datum/action/innate/cult/request_aid
+	name = "Request Assistance"
+	desc = "Request a number of cultists of your choosing to your location."
+	button_icon_state = "assistance"
+	check_flags = AB_CHECK_HANDS_BLOCKED|AB_CHECK_IMMOBILE|AB_CHECK_CONSCIOUS
+	var/cooldown = 0
```

Comment:
Use the cooldown defines.

Reply:
wdym by cooldown defines? I don't see anything for /datum/action or /datum/action/innate. There's a /datum/action/cooldown proc but none of the other cult stuff makes use of that.

Reply:
COOLDOWN_DECLARE, COOLDOWN_FINISHED, etc

Reply:
https://github.com/tgstation/tgstation/blob/76cee651a6480d846c07641df48794e6eedce1f0/code/__DEFINES/cooldowns.dm

Stuff found here